### ISSUE TITLE
Everyone receive message "sound" even they shouldn't in ver 2.5

### ISSUE THREAD

**Issue by mhtvsSFrpHdE** [montlikadani/AutoMessager#20]
### Problem

I've just updated from 2.4 to 2.5.  
Everyone receives message "sound" even they shouldn't in version 2.5  
Notify message is not printed though, just the sound,  
but **OP receives the message** and sound at the same time,  
they shouldn't either.

The config file is
```
messages:
permission:automessager.customMessage_Everyone receive the notify sound even they don't have this permission
group:customMessage_Same effect as above, even if player don't belong to this group
```

### Details
Plugin version: 
2.5

Software version: 
```
[INFO]: This server is running CraftBukkit version git-Spigot-79a30d7-acbc348 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)
[INFO]: This is a final build for 1.12.2. Please see https://www.spigotmc.org/ for details about upgrading.
```

Relevant plugins: 

### Console error
```
No console error or log message print.
```

### Configuration file(s)

config.yml & : announces.yml
https://pastebin.com/FySpH4EW

**Comment by montlikadani** [comment]
Yea fixed here https://github.com/montlikadani/AutoMessager/commit/7308fca37f9c4c71b29c7fc8de50df21bb51906d

**Comment by mhtvsSFrpHdE** [comment]
Still PermissionEX as https://github.com/montlikadani/AutoMessager/issues/18

The entire server config hosted at  
https://github.com/mhtvsSFrpHdE/WaterBottle-MinecraftServer-Spec

**Comment by mhtvsSFrpHdE** [comment]
> 
> 
> Yea fixed here [7308fca](https://github.com/montlikadani/AutoMessager/commit/7308fca37f9c4c71b29c7fc8de50df21bb51906d)

@montlikadani commented 18 second ago DEJAVU😂

**Comment by mhtvsSFrpHdE** [comment]
@montlikadani 2.6 with Minecraft 1.15.2 same issue appears again...
